### DISCUSSION TITLE
Trackpad detected as a joystick

### DISCUSSION THREAD

**Discussion by CubeOfCheese** [godotengine/godot-proposals#9892] [Editor]
Not sure where the source of this problem is so I'm starting a discussion instead of opening an issue

When I try to add an input in the Input Map editor, I hit an arrow key (example) and then when I try to move my mouse to hit okay, the event is detected as a joystick event. So I have to manually filter and select events.

I am running Godot 4.2.2.stable on Fedora KDE on a HP Probook.
![image](https://github.com/godotengine/godot-proposals/assets/16740469/c90e144a-ca5d-48e5-a1ab-85ae6658256a)

**Comment by Zireael07** [comment]
A bug (in SDL IIRC), not a discussion

**Comment by AThousandShips** [comment]
This is indeed a bug, and already tracked:
* https://github.com/godotengine/godot/issues/59250

Closing but thank you nonetheless
